src/.pylintrc.jinja

```diff
@@ -24,8 +24,20 @@
     no-utf8-coding-comment,
     odoo-addons-relative-import,
     old-api7-method-defined,
+{%- if odoo_version >= 18 %}
+    prefer-env-translation,
+{%- endif %}
     redefined-builtin,
     too-complex,
     unnecessary-utf8-coding-comment
```

Comment:
missing coma here,

might be better to put all of it befor redefined-builtin, isn't it?